### DISCUSSION TITLE
DynamicPermission

### DISCUSSION THREAD

**Discussion by khanhduatd186** [abpframework/abp#20209] [Q&A]
why when creating a new PermissionGroup does not show up the interface while adding Permission, please help me
![image](https://github.com/abpframework/abp/assets/127516595/924122ef-5b86-462b-bd48-0a2cd444e58c)
![image](https://github.com/abpframework/abp/assets/127516595/ce7b3ff2-bf34-492f-ad8b-37d9967a2a0a)

**Comment by maliming** [comment]
hi

You can check this question: https://support.abp.io/QA/Questions/5490/Need-to-refresh-the-list-of-static-permission-definitions-without-site-reload

**Reply by khanhduatd186** [reply]
When adding it that way, how do I make it show up on the interface I posted the question there?

**Reply by maliming** [reply]
See https://github.com/abpframework/abp-samples/tree/master/DynamicPermission

**Reply by khanhduatd186** [reply]
thanks
